src/core/create-ad-slot.ts

```diff
@@ -33,10 +33,8 @@ const adSlotConfigs: AdSlotConfigs = {
 		label: false,
 		refresh: false,
 	},
-	'high-merch': {
-		label: false,
+	'merchandising-high': {
 		refresh: false,
-		name: 'merchandising-high',
 	},
```

Comment:
This diff is a little hard to read but contains two changes:

- We remove `label: false` from the definition. This is uneccessary, since when a merch lands in a slot it doesn't get a label as it's fluid.
- Rename `high-merch` to `merchandising-high` for consistency. This allows us to omit the `name: 'merchandising-high` attribute as well.